CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration_TemplateMod.uc

```diff
@@ -360,10 +360,10 @@ static function PatchUtilityItems ()
 
 static function PatchItemStats()
 {
-	local X2ItemTemplateManager			TemplateManager;
-	local X2EquipmentTemplate			Template;
-	local InfiltrationModifier			InfilMod;
-	local name							ItemName;
+	local X2ItemTemplateManager		TemplateManager;
+	local X2EquipmentTemplate		Template;
+	local InfiltrationModifier		InfilMod;
+	local name						ItemName;
```

Comment:
The new version is better, but still weird...

Reply:
I think it's just github being github because it looks perfectly fine in modbuddy

Reply:
It's just a difference between using a tabstop of 4 in ModBuddy vs a tabstop of 8 on GitHub presumably.